### PR TITLE
feat(yoco): add new icons to Yoco

### PR DESCRIPTION
**Pull request by santiagoballadares**
Add icons:
- attention
- circle_magnifying_glass
- circle_minus
- magnifying_glass

- bump version to 2.6.1

### CODE REVIEW

File: packages/yoco/svg/magnifying_glass.svg
Comment (Isokaeder) [inline packages/yoco/svg/magnifying_glass.svg:1]: We already have an icon `search` that depicts a magnifying glass, this seems superfluous.

File: packages/yoco/svg/circle_magnifying_glass.svg
Comment (Isokaeder) [inline packages/yoco/svg/circle_magnifying_glass.svg:1]: - Should be called `circle_search`
- line width of "inner" aka the search/magnifying glass should be slightly smaller, looks weird next to other circle icons. See for example icon `circle_question`

**Review by Isokaeder — CHANGES_REQUESTED** [review]
IMO needs some small improvements.

Also consider making this a `2.7.0` release, since it adds new features.

**Comment by santiagoballadares** [comment]
Closed in favor of https://github.com/3YOURMIND/kotti/pull/964

**Pull request closed without merging**